### ISSUE TITLE
Clean up Dev branch

### ISSUE THREAD

**Issue by slucey** [NOAA-EDAB/Rpath#61]
Lots of stale branches that need to be dealt with below is a list of branches and the date of their latest commit:

- `sense_alpha`
- `lab_savepoint`
- `lab_563_spring2020` (includes the previous two branches) 9/2/20
- `master_R4` 6/3/20
- `adjust_vectors` 6/17/21
- `bioen` 8/3/21
- `gh-pages` 10/13/21
- `updateVignettes` 12/6/21
- `lab_563_Spring2022` 4/25/22

Last merge of dev to master 4/25/22

- `indiseas` 6/8/22
- `stanza_rollback` 7/11/22

Last dev commit 1/13/23

**Comment by slucey** [comment]
Looks like `bioen' was manually added back using the `bioen_merge` branch on 11/19/21

**Comment by slucey** [comment]
@kaydin I assume you want to leave the lab branches as a quick way to reference back to the version used in the class.

**Comment by kaydin** [comment]
@slucey yes I was going to suggest keeping the lab ones for reference, in case my class exercises stop working in future.  I think I manually moved/cleaned up a few functions from the labs to dev that were globally useful, so nothing needed from the labs that isn't already in dev.

**Comment by kaydin** [comment]
eco_data is only ahead of dev because a new library version forced an update of RcppExports.cpp.  Everything needed is merged into dev so safe to remove that one.

**Comment by slucey** [comment]
@kaydin Great!  Is there anything in `master_R4` that need to be merged into dev?

**Comment by slucey** [comment]
Merging `update_vignettes` via pull request #67

**Comment by slucey** [comment]
`stanza_rollback` is being address via issue #45

**Comment by slucey** [comment]
> @kaydin Great! Is there anything in master_R4 that need to be merged into dev?

@kaydin looks like the only thing here is an update to the version of Roxygen.  Is this more of a marker for you like the labs?

**Comment by slucey** [comment]
`adjust_vectors` was a minor tweak to `adjust.scenarios` that allows a user to specify a vector of groups to be changed.  Also included some NOAA branding for the pkgdown site.

**Comment by slucey** [comment]
After pulls #67 and #68 `dev` will be ready to merge into `master` and will change version number to 0.1.5.  Then we will be ready to pull `sense_beta` into dev and finish documentation issues.  Should quickly move `dev` with ecosense to `master` and move Rpath to version 0.2

**Comment by kaydin** [comment]
Sorry thought I'd replied earlier - master_R4 was a temporary bridge between R versions 3 and 4 due to some library issues during that transition.  Nothing here needed now.

**Comment by kaydin** [comment]
Did substantial group cleanup - still need to hand-check a few dangling ones.

**Comment by kaydin** [comment]
Clean enough now that we can keep things up to date as we go.
